src/SourceBuild/content/eng/Versions.props

```diff
@@ -25,13 +25,15 @@
     -->
     <PrivateSourceBuiltSdkVersion>9.0.100-preview.4.24179.1</PrivateSourceBuiltSdkVersion>
     <PrivateSourceBuiltArtifactsVersion>9.0.100-preview.4.24179.1</PrivateSourceBuiltArtifactsVersion>
-    <!-- msbuild -->
+    <!-- command-line-api dependencies -->
+    <SystemCommandLineVersion>2.0.0-beta4.24126.1</SystemCommandLineVersion>
+    <!-- msbuild dependencies -->
     <MicrosoftBuildVersion>17.8.3</MicrosoftBuildVersion>
-    <!-- runtime -->
+    <!-- runtime dependencies -->
     <MicrosoftExtensionsFileSystemGlobbingVersion>9.0.0-preview.2.24128.5</MicrosoftExtensionsFileSystemGlobbingVersion>
     <MicrosoftExtensionsLoggingConsoleVersion>9.0.0-preview.2.24128.5</MicrosoftExtensionsLoggingConsoleVersion>
     <MicrosoftExtensionsLoggingVersion>9.0.0-preview.2.24128.5</MicrosoftExtensionsLoggingVersion>
-    <!-- command-line-api -->
-    <SystemCommandLineVersion>2.0.0-beta4.24126.1</SystemCommandLineVersion>
```

Comment:
@MichaelSimons - In the context of https://github.com/dotnet/source-build/issues/4286, is this an example of where we would need to add an entry for `Newtonsoft Json` to Version.Details?

Reply:
I think we should get rid of Newtonsoft.Json as a dependency in the VMR orchestrator. There are just a handful of tasks that use it. Those should be converted to System.Text.Json.

Reply:
Agreed - tracked with https://github.com/orgs/dotnet/projects/101/views/1?filterQuery=newton&pane=issue&itemId=30084485